cluster/ci/config/prow/config.yaml

```diff
@@ -417,6 +417,24 @@ presubmits:
     rerun_command: "/test e2e_fedora"
     trigger: "((?m)^/test( all| fedora| e2e| e2e_fedora),?(\\s+|$))"
 
+  - name: test_pull_request_crio_critest_rhel
+    agent: jenkins
+    labels:
+      master: ci.openshift.redhat.com
+    always_run: false
```

Comment:
Why not always?

Reply:
Probably because the rest of the jobs in the repo are false, too. Why don't we run all of them continuously by default?

Reply:
We could do that, I'll make sure to follow up after this is in